jetcd-core/src/test/java/com/coreos/jetcd/internal/impl/SslTest.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2017 The jetcd authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.coreos.jetcd.internal.impl;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import com.coreos.jetcd.Client;
+import com.coreos.jetcd.KV;
+import com.coreos.jetcd.data.ByteSequence;
+import io.grpc.netty.GrpcSslContexts;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Objects;
+import org.junit.Test;
+
+public class SslTest {
+
+  @Test(timeout = 5000)
+  public void testSimpleSllSetup() throws Exception {
+    final ByteSequence key = ByteSequence.fromString(TestUtil.randomString());
+    final ByteSequence val = ByteSequence.fromString(TestUtil.randomString());
+    final String capath = System.getProperty("ssl.cert.capath");
+    final String authority = System.getProperty("ssl.cert.authority", "etcd-ssl");
+    final String endpoints = System.getProperty("ssl.cert.endpoints", "https://127.0.0.1:42379");
+
+    try (InputStream is = Objects.nonNull(capath)
+          ? new FileInputStream(new File(capath))
+          : getClass().getResourceAsStream("/ssl/cert/ca.pem")) {
```

Comment:
I think we are not using property to get the capth anymore. maybe just `getClass().getResourceAsStream("/ssl/cert/ca.pem")`? will be enough?

Reply:
I want to be able to test against my own etcd setup with my own ca, so this is still needed